### DISCUSSION TITLE
Error oppening configuration file

### DISCUSSION THREAD

**Discussion by rabonarca** [qdm12/gluetun#681] [Q&A] [answered]
Hello,

I have an issue with the .ovpn file. 
I have the error  : no such file or directory, but the file is here.

Can someone help me please ?

![image](https://user-images.githubusercontent.com/48682896/137833497-e5656c54-eea1-4751-b819-17c9afa102d1.png)
![image](https://user-images.githubusercontent.com/48682896/137834316-7a877dba-423b-433c-a4e5-4f38d1eff535.png)

Regards

**Comment by qdm12** [comment, marked as answer]
`target.ovpn` is not a valid path (as in its relative to where the gluetun binary is). I would recommend you to follow https://github.com/qdm12/gluetun/wiki/Custom-provider#tldr and leave that variable empty (default).

**Reply by rabonarca** [reply]
Thank you for your quick answer. It worked !
